src/configs/.env

```diff
@@ -4,4 +4,5 @@ API_PREFIX='/'
 LOG_FILE_PATH='./src/logs'
 MongoURI='mongodb://localhost:27017'
 GITLAB_READ_API='https://gitlab.com/api/v4/projects/4600360/repository/files'
-GITLAB_TAGS_API='https://gitlab.com/api/v4/projects/4600360/repository/tags'
\ No newline at end of file
+GITLAB_TAGS_API='https://gitlab.com/api/v4/projects/4600360/repository/tags'
+GITLAP_HOOK_TOKEN='OURGITLABTOKENHERE'
```

Comment:
Should we be tracking this file?

Reply:
Also there's a typo, it says GITLAP

Reply:
Shouldn't be tracked, resolved and the type is fixed